src/tests/ftest/soak/soak.py

```diff
@@ -222,22 +226,37 @@ def get_remote_logs(self):
         # copy the files from the remote
         # TO-DO: change scp
         this_host = socket.gethostname()
-        rsync_str = "rsync -avtr --min-size=1B"
+        cmd = [
+            "bash", "-c",
```

Comment:
Do we need full paths for the `bash` and `rsync` commands?  I don't see Bandit complaining about it in the TestResults.

Reply:
not needed.